python/tank/bootstrap/import_handler.py

```diff
@@ -174,7 +174,7 @@ def _swap_core(self, core_path):
 
             # sort by package depth, deeper modules first
             module_names = sorted(
-                sys.modules.keys(),
+                list(sys.modules.keys()),
```

Comment:
No need to call list here, sorted can deal with iterators.

```
>>> a = {9:2, 3:4, 1:6, 7:8, 4:0}
>>> sorted(a.keys())
[1, 3, 5, 7, 9]
```

Reply:
✅